### PR TITLE
include path reading into reader manager

### PR DESCRIPTION
**Pull request by GregorCH**

### CODE REVIEW

File: ipet/parsing/ReaderManager.py [hunk 1]
```diff
@@ -12,7 +12,7 @@
 import os
 import logging
 import xml.etree.ElementTree as ElementTree
-from .StatisticReader import ErrorFileReader, GapReader, TimeLimitReader, PathReader, ListReader, \
```
Comment (fschloesser) [inline ipet/parsing/ReaderManager.py]: We should leave the PathReader in

File: ipet/parsing/ReaderManager.py [hunk 2]
```diff
@@ -175,7 +175,6 @@ def registerDefaultReaders(self):
              MetaDataReader(),
              NodesReader(),
              ObjsenseReader(),
-             PathReader(),
```
Comment (fschloesser) [inline ipet/parsing/ReaderManager.py]: leave it in here as well

File: ipet/parsing/StatisticReader.py [hunk 3]
```diff
@@ -193,23 +193,6 @@ def extractStatistic(self, line):
                 self.addData(key, timestamp)
                 break
 
-class PathReader(StatisticReader):
```
Comment (fschloesser) [inline ipet/parsing/StatisticReader.py]: leave this in

Comment (GregorCH) [inline ipet/parsing/StatisticReader.py]: Why?

Comment (fschloesser) [inline ipet/parsing/StatisticReader.py]: Why not?

Comment (GregorCH) [inline ipet/parsing/StatisticReader.py]: Note that "@01" is a modifiable argument in the reader manager, and we parse it now in the reader manager. This way, we are both faster, use less code, and are more flexible.

**Pull request merged**